src/pages/proposals/ProposalList.vue

```diff
@@ -506,11 +512,37 @@ q-page.page-proposals
         icon= "fas fa-file-medical" :actionButtons="[{label: 'Create a new Proposal', color: 'primary', onClick: () => $router.push(`/${this.daoSettings.url}/proposals/create`), disable: !isMember, disableTooltip: 'You must be a member'}]" )
       div(v-if="!filteredProposals.length && !filteredStagedProposals.length" class="row justify-center q-my-md")
         loading-spinner(color="primary" size="72px")
+      .row.q-my-md
+        .h-h3 Staging proposals
+        .h-h3.q-ml-xs.proposal-amount ({{ proposalsCount.stagingpropAggregate.count }})
```

Comment:
@pugakn `proposalsCount.stagingpropAggregate.count` is raw response from GraphQL. Would be much better if in template we would have just value like `{{proposalsCount}}`, we shouldn't leak our implementation details in template. 

Formatting the data in format that our app needs should be done at the data fetching levels.

This would ensure that in case that GraphQL response changes, we need to update it just on one place, and not like now all over the app.